scripts/globals/mobskills/gravity_field.lua

```diff
@@ -13,7 +13,11 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    skill:setMsg(xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.SLOW, 4500, 0, math.random(240, 420)))
+    if target:hasStatusEffect(xi.effect.HASTE) then
+        return
```

Comment:
What does the messaging look like when this happens? Is it correct?

Reply:
I recently ran into this also with stygen vapor and the other one. If no message is there it usually defaults to KO message.